bosshud/ze_castle_bridge_v1_4.txt

```diff
@@ -0,0 +1,13 @@
+"math_counter"
+{
+    "config"
+    {
+        "BossBeatenShowTopDamage"   "1"
+    }
```

Comment:
You only need to include config values if you're changing them from their default value.